vane-portals/src/main/resources/plugin.yml

```diff
@@ -6,10 +6,10 @@ authors: [oddlama]
 website: 'https://github.com/oddlama/vane'
 
 depend: [vane-core]
-softdepend: [dynmap, BlueMap]
+softdepend: [dynmap, BlueMap, Pl3xMap]
 
 main: org.oddlama.vane.portals.Portals
 database: false
 
 api-version: 1.19
-load: STARTUP
```

Comment:
I had to revert this change, as it causes portals to not be loaded at startup. Does that affect your Plexmap logic in any way?

Reply:
Been awhile since I've looked at this, don't remember why this was necessary. I'll test it out and try to work around it when I can.

Reply:
Thanks, it's not urgent so take your time :)